src/shared/components/page-layout/index.js

```diff
@@ -0,0 +1,3 @@
+import PageLayout from './PageLayout';
```

Comment:
Nifty pick: can we do `export { default } from './PageLayout`? Check in other files of this PR as well.